### PR TITLE
Add new blog post about expired GPG and newest 1.6 release

### PR DESCRIPTION
**Pull request by zetok**
cc: @Diadlo , @initramfs , @sudden6

---
This change is [Reviewable](https://reviewable.io/reviews/qtox/blog/4)

### CODE REVIEW

**Comment by initramfs** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 18 at r1](https://reviewable.io:443/reviews/qtox/blog/4#-KWUWnBBt0qoz03PzVli:-KWUWnBBt0qoz03PzVlj:b-g8ei0v) ([raw file](https://github.com/qtox/blog/blob/c5de7a28cf1205b270137e2a0568392b6d05bd9a/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L18)):_

> ``` Markdown
> problem with the [expired GPG key] used to sign DEB packages.
> 
> Problem was quickly fixed, but not without casualties. Meaning, manual work on
> ```

```
Problem was quickly fixed, but not without casualties.
```

to

```
The problem was quickly fixed, but not without some casualties.
```

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 18 at r1](https://reviewable.io:443/reviews/qtox/blog/4#-KWUX5en0ykfB7kSsNQY:-KWUX5en0ykfB7kSsNQZ:b-3m2p96) ([raw file](https://github.com/qtox/blog/blob/c5de7a28cf1205b270137e2a0568392b6d05bd9a/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L18)):_

> ``` Markdown
> problem with the [expired GPG key] used to sign DEB packages.
> 
> Problem was quickly fixed, but not without casualties. Meaning, manual work on
> ```

```
Meaning, manual work on your side will be required.
```

Get rid of the "Meaning", so just:

```
Manual work on your side will be required.
```

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 45 at r1](https://reviewable.io:443/reviews/qtox/blog/4#-KWUXnv2d44aiTIyFYeO:-KWUXnv2d44aiTIyFYeP:b-xyrplw) ([raw file](https://github.com/qtox/blog/blob/c5de7a28cf1205b270137e2a0568392b6d05bd9a/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L45)):_

> ``` Markdown
> #### Auto-answer calls
> 
> If you want to automatically pick up video or audio call from selected
> ```

```
If you want to automatically pick up video or audio call from selected contacts, now you can.
```

Move "now you can" to the start:

```
Now you can automatically pick up video or audio call from selected contacts.
```

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 53 at r1](https://reviewable.io:443/reviews/qtox/blog/4#-KWUY69HupUi2UfFZpH8:-KWUY69IjAcgPCMVOTBn:b-n1lpya) ([raw file](https://github.com/qtox/blog/blob/c5de7a28cf1205b270137e2a0568392b6d05bd9a/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L53)):_

> ``` Markdown
> you have any ideas how else they could be made better, please [make an issue]
> with a proposal. This might sound a bit scary/bothersome, but it really isn't.
> Even a screenshot with added arrows can be good enough, provided that there is
> ```

```
provided that there is attached some reasoning about the change
```

change to

```
provided that reasoning about the change is included.
```

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 66 at r1](https://reviewable.io:443/reviews/qtox/blog/4#-KWUYZkfqDnWNzvUO31_:-KWUYZkfqDnWNzvUO31a:biwrr2) ([raw file](https://github.com/qtox/blog/blob/c5de7a28cf1205b270137e2a0568392b6d05bd9a/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L66)):_

> ``` Markdown
> Full [changelog here].
> 
> We hope that you will enjoy :-)
> ```

Probably better to shorten this to just:

```
Enjoy :-)
```

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/blog/4)_

**Comment by zetok** [comment]
Review status: 0 of 1 files reviewed at latest revision, 5 unresolved discussions.

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 18 at r1](https://reviewable.io:443/reviews/qtox/blog/4#-KWUWnBBt0qoz03PzVli:-KWU_4bPg8gIuBxq0IFK:b-896fix) ([raw file](https://github.com/qtox/blog/blob/c5de7a28cf1205b270137e2a0568392b6d05bd9a/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L18)):_
<details><summary>

<i>Previously, initramfs wrote…</i></summary>



> ```
> Problem was quickly fixed, but not without casualties.
> ```
> 
> to
> 
> ```
> The problem was quickly fixed, but not without some casualties.
> ```
> 
> </details>
> Done.

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 18 at r1](https://reviewable.io:443/reviews/qtox/blog/4#-KWUX5en0ykfB7kSsNQY:-KWU_6wYSn7q41ENTYEc:b-896fix) ([raw file](https://github.com/qtox/blog/blob/c5de7a28cf1205b270137e2a0568392b6d05bd9a/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L18)):_
<details><summary>

<i>Previously, initramfs wrote…</i></summary>



> ```
> Meaning, manual work on your side will be required.
> ```
> 
> Get rid of the "Meaning", so just:
> 
> ```
> Manual work on your side will be required.
> ```
> 
> </details>
> Done.

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 45 at r1](https://reviewable.io:443/reviews/qtox/blog/4#-KWUXnv2d44aiTIyFYeO:-KWU_Ah2zDWvgS8ZDTMW:b-896fix) ([raw file](https://github.com/qtox/blog/blob/c5de7a28cf1205b270137e2a0568392b6d05bd9a/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L45)):_
<details><summary>

<i>Previously, initramfs wrote…</i></summary>



> ```
> If you want to automatically pick up video or audio call from selected contacts, now you can.
> ```
> 
> Move "now you can" to the start:
> 
> ```
> Now you can automatically pick up video or audio call from selected contacts.
> ```
> 
> </details>
> Done.

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 53 at r1](https://reviewable.io:443/reviews/qtox/blog/4#-KWUY69HupUi2UfFZpH8:-KWU_HcuhFWiEDo-elap:b-g07ezx) ([raw file](https://github.com/qtox/blog/blob/c5de7a28cf1205b270137e2a0568392b6d05bd9a/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L53)):_
<details><summary>

<i>Previously, initramfs wrote…</i></summary>



> ```
> provided that there is attached some reasoning about the change
> ```
> 
> change to
> 
> ```
> provided that reasoning about the change is included.
> ```
> 
> </details>
> ~Done.

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 66 at r1](https://reviewable.io:443/reviews/qtox/blog/4#-KWUYZkfqDnWNzvUO31_:-KWU_H9fBmld5D0On_Vs:b-896fix) ([raw file](https://github.com/qtox/blog/blob/c5de7a28cf1205b270137e2a0568392b6d05bd9a/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L66)):_
<details><summary>

<i>Previously, initramfs wrote…</i></summary>



> Probably better to shorten this to just:
> 
> ```
> Enjoy :-)
> ```
> 
> </details>
> Done.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/blog/4)_

**Comment by initramfs** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 57 at r2](https://reviewable.io:443/reviews/qtox/blog/4#-KWUaIm8zNTOLzNt9EhY:-KWUaIm9pplcVMdZYXR0:b9c421k) ([raw file](https://github.com/qtox/blog/blob/e17fb76324c17e1db846c5527f71c38b3315728d/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L57)):_

> ``` Markdown
> #### Breaking changes
> 
> None (that we know of). Testing week mentioned in the previous blog post passed
> ```

```
... previous blog post passed away without any ...
```

removed to away:

```
... previous blog post passed without any ...
```

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 61 at r2](https://reviewable.io:443/reviews/qtox/blog/4#-KWUaYy2BSXQClVIVLRk:-KWUaYy3dR1sapBx4557:b-dv3huu) ([raw file](https://github.com/qtox/blog/blob/e17fb76324c17e1db846c5527f71c38b3315728d/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L61)):_

> ``` Markdown
> anything surprising out there.
> 
> #### Other
> ```

Make the category name "Change log"? Since it's essentially linking to a change log.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/blog/4)_

**Comment by zetok** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

_[_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md, line 61 at r2](https://reviewable.io:443/reviews/qtox/blog/4#-KWUaYy2BSXQClVIVLRk:-KWUee5e1wwibC5KIyRz:b-896fix) ([raw file](https://github.com/qtox/blog/blob/e17fb76324c17e1db846c5527f71c38b3315728d/_posts/2016-11-13-Expired-GPG-and-v1.6.0-release.md#L61)):_
<details><summary>

<i>Previously, initramfs wrote…</i></summary>



> Make the category name "Change log"? Since it's essentially linking to a change log.
> </details>
> Done.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/blog/4)_

**Comment by initramfs** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/blog/4#-:-KWUfXcmnO6Wym0sGZI6:bnfp4nl)_
